django/cantusdb_project/main_app/management/commands/update_cached_concordances.py

```diff
@@ -62,7 +55,6 @@ def get_concordances() -> dict:
         dict: A dictionary where each key is a Cantus ID and each value is a list all
```

Comment:
The doctring for this function is out of date since there is now no more grouping....